jetstream/mr2022-backgroundtaskmessage-notification-release-2pct-with-holdback.toml

```diff
@@ -0,0 +1,211 @@
+## Background messaging tasks present several wrinkles compared to regular
+## Firefox in-product (browsing) experiments.  See [Jetstream support for
+## background tasks Firefox messaging system
+## experiments](https://docs.google.com/document/d/1GeOGUmG8XpitdexW2HESZJbCNhTT-o6qe1MmBsDPDHE/edit)
+## for an overview.
+##
+## This specific analysis uses legacy client IDs as its single unique
+## identifier, answering the main open question of the document linked above.
+##
+## This analysis is complicated by being written to be more general than this
+## single experiment; it tries to accommodate future experiments by referencing
+## the experiment slug indirectly and to handle in-product changes that will
+## apply to future experiments but do not apply to this one.
+##
+## For technical reasons, we take the union of multiple queries to determine
+## enrollment; see the extensive inline comments below.  It is possible to
+## segment by the various sub-queries, but it is awkward and does not provide
+## much value (beyond exposing counts of the various segments, which is valuable
+## during development but not relevant to impact analysis).
+##
+## Note that background update telemetry can be delayed, and that re-engaging
+## with Firefox can delay that telemetry further since the background update
+## task does not run if Firefox itself is running.  Therefore, even though the
+## true enrollment period -- the time when Nimbus is enrolling clients via
+## Remote Settings -- is generally *one* week, we artificially expand the
+## enrollment period to *two* weeks for Jetstream's analysis.
+
+[experiment]
+
+start_date = "2023-01-23"
+end_date = "2023-02-20"
+enrollment_period = 14
+
```

Comment:
I think we should add `reference_branch = "disabled"` here. Right now, the experiment is configured with `enabled` as the control branch.

Reply:
I've done this.  Should my reference branch have been named something else to have this automatically?  `"control"`?

Reply:
Follow-up: why isn't this determined from the Experimenter recipe, which I think knows about reference/control branches?

Reply:
It is determined automatically from Experimenter and doesn't need to be named anything in particular. It's done as a part of branch setup in Experimenter: the first branch is marked as the control branch so that's where we could put the `disabled` branch. The `enabled` branch could go next. In this case I think the branches in Experimenter got swapped, so that's why we need the manual override. 
![Screenshot 2023-02-06 at 9 28 44 AM](https://user-images.githubusercontent.com/2738595/217042373-c4b2d773-7a78-40c8-b273-2bd63804889b.png)